resources/js/features/home/components/+root/TrendingRightNow/TrendingRightNow.tsx

```diff
@@ -20,9 +20,15 @@ export const TrendingRightNow: FC = () => {
       <div className="grid gap-1 sm:grid-cols-2">
         {trendingGames.map((trendingGame) => (
           <div key={`trending-${trendingGame.game.id}`} className="rounded-lg bg-embed p-2">
-            <div className="flex w-full items-end justify-between">
-              <GameAvatar {...trendingGame.game} size={40} showSystemChip={true} />
-              <p className="text-2xs">
+            <div className="relative flex w-full items-end justify-between">
+              <GameAvatar
+                {...trendingGame.game}
+                size={40}
+                showSystemChip={true}
+                gameTitleClassName="line-clamp-1"
```

Comment:
Not sure how it determines where to do the truncation, but I find it weird that it'll turn the last two characters of a word into "...", but one more is too much and it just nukes the word entirely.
![image](https://github.com/user-attachments/assets/4d86e524-dd3f-4327-bd32-5ea22f84d15a)
![image](https://github.com/user-attachments/assets/a186f762-fdc0-4398-938d-4166a9017453)

Reply:
For better or worse, this one is at the mercy of the browser vendor's CSS implementation. One benefit of doing it with a `line-clamp-*` classname though is we don't have to pull in a dependency to do the truncation based on the div width.